Tests/UnitTests/ReportOrderBodyTests.swift

```diff
@@ -71,7 +71,8 @@ class ReportOrderBodyTests: XCTestCase {
         
         // Assert
         XCTAssertEqual(body.dictionaryRepresentation as NSDictionary,
-                       ["btn_ref": "srctok-abc123",
+                       ["advertising_id": "00000000-0000-0000-0000-000000000000",
```

Comment:
I'd like to see a test for when the `advertising_id` is disabled asserting that the payload omits the field.